### PR TITLE
Enable the leading/trailing whitespace-sensitive options.

### PR DESCRIPTION
**Pull request by ltratt**
Amusingly, these required no meaningful changes. I should have enabled them earlier!

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -147,13 +147,15 @@ impl<'a> FMBuilder<'a> {
         self
     }
 
-    #[doc(hidden)]
+    /// If `yes`, then leading whitespace will be ignored in both pattern and text; otherwise
+    /// leading whitespace must match between the two. Defaults to `true`.
```
Comment (vext01) [inline src/lib.rs]: "match between the two" sounds odd.

How about:
> otherwise leading whitespace must be matched.

Comment (ltratt) [reply src/lib.rs]: How about just "must match"?

Comment (ltratt) [reply src/lib.rs]: Sorry, as in https://github.com/softdevteam/fm/pull/14/commits/4637022d3629b8021a8e9b599fd668923086eb66.

File: src/lib.rs [hunk 2]
```diff
@@ -748,4 +746,28 @@ Text (error at line 3):
...
+
+        assert!(helper("a\na", "a\na"));
+
+        assert!(helper("a\n a", "a\n a"));
+        assert!(!helper("a\n a", "a\na"));
+        assert!(!helper("a\na", "a\n a"));
+
+        assert!(helper("a\na ", "a\na "));
+        assert!(!helper("a\na", "a\na "));
+        assert!(!helper("a\na ", "a\na"));
```
Comment (vext01) [inline src/lib.rs:771]: None of these test inputs have leading whitespace?

Or is it leading whitespace within each line?

Comment (ltratt) [reply src/lib.rs:771]: Within each line.

Comment (ltratt) [reply src/lib.rs:771]: e.g. this test:

```
        assert!(!helper("a\n a", "a\na"));
```

Comment (vext01) [reply src/lib.rs:771]: I see. I think we should make that clearer that it's per-line. I thought we were talking about leading/trailing whitespace at the top/bottom of the entire test input.

Comment (ltratt) [reply src/lib.rs:771]: Agreed. Fixed in https://github.com/softdevteam/fm/pull/14/commits/7f2fac8568c0badfdf4f4bd359a6bb3f2565bc75.

**Comment by vext01** [comment]
Thanks for that!

Please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**